cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/cloudformation/ListStacksAction.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * (c) Copyright 2017 EntIT Software LLC, a Micro Focus company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.cloudslang.content.amazon.actions.cloudformation;
+
+import com.amazonaws.services.cloudformation.AmazonCloudFormation;
+import com.amazonaws.services.cloudformation.model.DescribeStackResourcesRequest;
+import com.amazonaws.services.cloudformation.model.DescribeStacksRequest;
+import com.amazonaws.services.cloudformation.model.Stack;
+import com.amazonaws.services.cloudformation.model.StackResource;
+import com.hp.oo.sdk.content.annotations.Action;
+import com.hp.oo.sdk.content.annotations.Output;
+import com.hp.oo.sdk.content.annotations.Param;
+import com.hp.oo.sdk.content.annotations.Response;
+import com.hp.oo.sdk.content.plugin.ActionMetadata.MatchType;
+import com.hp.oo.sdk.content.plugin.ActionMetadata.ResponseType;
+import io.cloudslang.content.amazon.entities.constants.Outputs;
+import io.cloudslang.content.amazon.utils.DefaultValues;
+import io.cloudslang.content.utils.OutputUtilities;
+
+import java.util.Map;
+
+import static io.cloudslang.content.amazon.entities.constants.Inputs.CommonInputs.*;
+import static org.apache.commons.lang3.StringUtils.*;
```

Comment:
Same comment here... especially when third party library is involved. It doesn't make sense to import it all... just method / methods needed. Minimal I mean... :smile: